src/status_im2/common/home/view.cljs

```diff
@@ -97,7 +97,10 @@
         (assoc button-common-props :accessibility-label :open-search-button)
         :i/search])
      [quo/button
-      (assoc button-common-props :accessibility-label :open-scanner-button)
+      (merge button-common-props
```

Comment:
you can use `assoc`, as it's multivariadic, and it's better suited when you have a known set of keys:

```
(assoc button-common-props
   :accessibility-label :open...
   :on-press                 #(disp..)}
```
Also trailing parentheses https://guide.clojure.style/#gather-trailing-parens